v1.go

```diff
@@ -22,17 +23,20 @@ func initVersion1() *Header {
 }
 
 func parseVersion1(reader *bufio.Reader) (*Header, error) {
-	// Read until LF shows up, otherwise fail.
-	// At this point, can't be sure CR precedes LF which will be validated next.
-	line, err := reader.ReadString('\n')
-	if err != nil {
+	// The header must be terminated by CRLF and cannot be more than 107 bytes long.
```

Comment:
Please, add a note here justifying the _107 bytes long_, such as per spec
> (...) a 108-byte buffer is always enough to store all the line and a trailing zero for string processing.

Reply:
ok